dist/api_data.py

```diff
@@ -239,6 +239,17 @@ def __str__(self):
                             the number of triggers for which to wait''',
                             min='1', max='4294967295'),
                 ]),
+            ConnectionControlPoint('thread_wait_for_upd_abort', 'Trigger', 'Always', '', 
+                r'''Thread waits for transaction to get aborted.''',
+                type='category', subconfig= [
+                    # Action configuration parameters
+                    Config('wait_count', '1', r'''
+                            the number of triggers for which to wait''',
+                            min='1', max='4294967295'),
+                    Config('enable_count', '1', r'''
+                        the number of triggers for which to wait''',
```

Comment:
In WT- added the config parameters as comments for predicates to src/cp/cp_pred.c and for actions to src/include/cp_action.h.
New line 250 should be the same as cp_pred.c line 76.